src/async-wrap.cc

```diff
@@ -448,6 +451,56 @@ static void DisablePromiseHook(const FunctionCallbackInfo<Value>& args) {
   }, static_cast<void*>(env));
 }
 
+static void AsyncTaskScheduled(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  if (!args[0]->IsString())
+    return env->ThrowTypeError("first argument must be a string");
+  if (!args[1]->IsNumber())
+    return env->ThrowTypeError("second argument must be a number");
+  if (!args[2]->IsBoolean())
+    return env->ThrowTypeError("third argument must be a boolean");
+
+  Local<String> taskName = args[0]->ToString(env->context()).ToLocalChecked();
```

Comment:
Since you already checked `args[0]->IsString()`, there’s no need for an explicit cast here; you can just use `taskName = args[0].As<String>()`.